WindowsServerDocs/virtualization/hyper-v/Best-Practices-for-running-Linux-on-Hyper-V.md

```diff
@@ -72,6 +72,8 @@ Configure and use the virtual Ethernet adapter, which is a Hyper-V-specific netw
 
 The Linux kernel has four different I/O schedulers to reorder requests with different algorithms. NOOP is a first-in first-out queue that passes the schedule decision to be made by the hypervisor. It is recommended to use NOOP as the scheduler when running Linux virtual machine on Hyper-V. To change the scheduler for a specific device, in the boot loader's configuration (/etc/grub.conf, for example), add **elevator=noop** to the kernel parameters, and then restart.
 
+On Ubuntu 18.04 with the Azure-tuned kernel uses multi-queue I/O schedulers, so "none" is the appropriate selection instead of "noop". See [Ubuntu I/O Schedulers](https://wiki.ubuntu.com/Kernel/Reference/IOSchedulers) for more information.
```

Comment:
The start of this sentence is slightly awkward in English. I recommend this minor edit:
```suggestion
Ubuntu 18.04 with the Azure-tuned kernel uses multi-queue I/O schedulers, so "none" is the appropriate selection instead of "noop". See [Ubuntu I/O Schedulers](https://wiki.ubuntu.com/Kernel/Reference/IOSchedulers) for more information.
```
If my suggestion changes the meaning of the sentence incorrectly, please suggest a better way to point out that this is about the Ubuntu version where the Azure-tuned kernel uses multi-queue I/O schedulers.